maro/rl/policy_v2/maac.py

```diff
@@ -239,7 +239,7 @@ def get_batch_loss(self, batch: dict, explicit_grad: bool = False) -> dict:
     def data_parallel(self, *args, **kwargs) -> None:
```

Comment:
Maybe we should remove `data_parallel()` here?

Reply:
I leave all the changes of `maac.py` in another pull request since this one is temporary.